File: app/src/main/java/com/summer/itis/summerproject/model/pojo/opensearch/SearchSuggestion.java
package com.summer.itis.summerproject.model.pojo.opensearch;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Element;

public class SearchSuggestion
{
    @Element
    private Section Section;

    @Element
    private String Query;

    @Attribute(required = false)
    private String xmlns;

    @Attribute
    private String version;

    public Section getSection ()
    {
        return Section;
    }

    public void setSection (Section Section)
    {
        this.Section = Section;
    }

    public String getQuery() {
        return Query;
    }

    public void setQuery(String query) {
        Query = query;
    }

    public String getXmlns ()
    {
        return xmlns;
    }

    public void setXmlns (String xmlns)
    {
        this.xmlns = xmlns;
    }

    public String getVersion ()
    {
        return version;
    }

    public void setVersion (String version)
    {
        this.version = version;
    }

    @Override
    public String toString()
    {
        return "ClassPojo [Section = "+Section+", Query = "+Query+", xmlns = "+xmlns+", version = "+version+"]";
    }
}

File: app/src/main/java/com/summer/itis/summerproject/model/pojo/opensearch/SearchSuggestion.kt
package com.summer.itis.summerproject.model.pojo.opensearch

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Element

class SearchSuggestion {
    @Element
    var section: Section? = null

    @Element
    var query: String? = null

    @Attribute(required = false)
    var xmlns: String? = null

    @Attribute
    var version: String? = null

    override fun toString(): String {
        return "ClassPojo [Section = $section, Query = $query, xmlns = $xmlns, version = $version]"
    }
}
